cg/meta/workflow/nf_analysis.py

```diff
@@ -803,3 +835,68 @@ def get_cases_to_store(self) -> list[Case]:
             if self.trailblazer_api.is_latest_analysis_completed(case_id=case.internal_id)
             or self.trailblazer_api.is_latest_analysis_qc(case_id=case.internal_id)
         ]
+
+    def get_reference_genome(self, case_id: str) -> GenomeVersion:
+        """Return reference genome for a case.
+        If this information is not available in StatusDB, it defaults to hg38."""
+        reference_genome: set[str] = {
+            sample.reference_genome
+            for sample in self.status_db.get_samples_by_case_id(case_id=case_id)
+        }
+        if len(reference_genome) == 1:
+            return reference_genome.pop()
+        elif len(reference_genome) > 1:
+            raise CgError(
+                f"Samples linked to case {case_id} have different reference genome versions set"
+            )
+        else:
+            LOG.info(f"No reference genome specified, defaulting to {GenomeVersion.hg38}")
+
+    def get_gene_panel_genome_build(self, case_id: str) -> GenePanelGenomeBuild:
+        """Return build version of the gene panel for a case."""
+        reference_genome = self.get_reference_genome(case_id=case_id)
+        try:
+            return getattr(GenePanelGenomeBuild, reference_genome)
+        except AttributeError as error:
+            LOG.error(
+                f"Reference {reference_genome} has no associated genome build for panels: {error}"
+            )
+
+    def get_gene_panel(self, case_id: str, dry_run: bool = False) -> list[str]:
```

Comment:
I would use directly the `get_gene_panel` function from the analysis API

Reply:
there is no defined get_gene_panel in analysis API, are you suggesting that I move it there instead?

Reply:
I meant to use directly `_get_gene_panel`, but I see now that we're using it from the base calls.

Could you then just define it in the analysis api and raise there a not implemented error so we fix the references 🙏

Reply:
done